### PR TITLE
🔥 💄 change style attributes for right view and add _app.tsx

### LINKED ISSUE #54: build arrows in picture

### PR DESCRIPTION
**Pull request by mewiemers**
- set `overflow`
- change `line-height` in different `.module.css`
- change `margin` and `padding` in different `css` files
- give `audio` a class
- change different `font-size`
- changed css structure from `grid` to `position`
- add `_app.tsx for `title`and `favicon`

### CODE REVIEW

**Review by mrana17 — APPROVED** [review]
You are an Expert!!! 👩‍🎓

File: pages/index.tsx [hunk 1]
```diff
@@ -1,15 +1,9 @@
-import Head from "next/head";
 import Logo from "../components/welcome/Welcome";
 import Link from "next/link";
 
 export default function Home() {
   return (
     <>
-      <Head>
-        <title>Meet Albert</title>
-        <link rel="icon" href="/albert-hey.ico" />
```
Comment (DNaunin) [inline pages/index.tsx:10]: It would be better if your favicon would be called favicon.ico, so everybody knows this is your favicon file.

**Review by DNaunin — APPROVED** [review]
Great Job! 🚀

**Comment by colefriedlaender** [comment]
Good job!

**Review by HorstFrank — APPROVED** [review]
LG!

**Pull request merged**